res/css/views/elements/_MessageEditor.scss

```diff
@@ -40,13 +40,42 @@ limitations under the License.
             color: white;
         }
 
+        span.room-pill {
+            color: $accent-fg-color;
+            background-color: $rte-room-pill-color;
+        }
+
+        span.user-pill {
+            color: $primary-fg-color;
+            background-color: $other-user-pill-bg-color;
+        }
+
         span.user-pill, span.room-pill {
```

Comment:
Are these styles copied from normal pills? Could we share more of the CSS so it's easier to change going forward?